Intersect.Client.Framework/Database/GameDatabase.cs

```diff
@@ -57,6 +57,8 @@ public abstract partial class GameDatabase
 
         public bool ShowManaAsPercentage { get; set; }
 
+        public bool TypewriterText { get; set; }
```

Comment:
Can you change this to `TyperwriterBehavior` and create an enum (of the same name) with two options right now, `Off` (default/0), and `Word` (at least I think you have it set to split per word not letter? If I am mistaken and it is per letter call it `Letter` instead)

I would prefer we not have to port this to an enum in the future, and I can definitely envision more behavior types that exceed a boolean.